include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
Maybe `CHEATING_SIZEOF()` would be more appropriate :D. Pretending there's data 8 bits wide when there isn't ;-)